sphinx/themes/bizstyle/layout.html

```diff
@@ -14,11 +14,6 @@
     <script src="{{ pathto('_static/bizstyle.js', 1) }}"></script>
 {%- endblock %}
 
-{# doctype override #}
-{%- block doctype %}
-<!doctype html>
```

Comment:
This was overriding `<!DOCTYPE html>`, unless I'm missing something this was incorrect and unecessary?

Reply:
Hmm.  A [code search for `block doctype`](https://github.com/search?q=path%3A*.py%20%22block%20doctype%22&type=code) finds a handful of cases -- mostly [`zato`](https://github.com/zatosource/zato/)-derived -- that still override this template block.

For example, here: https://github.com/zatosource/zato/blob/683ae439dcfd6056d43474abcaafaf96d3956d9a/code/zato-cli/src/zato/cli/_apispec_default.py#L194-L196

However.. that's [not using](https://github.com/zatosource/zato/blob/683ae439dcfd6056d43474abcaafaf96d3956d9a/code/zato-cli/src/zato/cli/_apispec_default.py#L566) the `bizstyle` theme.

I think what's going on here is that Sphinx _used_ to provide this block in the templates - it's [in v2.3.0 (chosen fairly arbitrarily) in the `basic` theme](https://github.com/sphinx-doc/sphinx/blob/86143a54b86049dc8d725ee1418e9523c41a7b77/sphinx/themes/basic/layout.html#L10-L12), for example.

Reply:
This was probably for HTML 4 specs and before where you needed to include a DTD reference.